content/upgrades/index.md

```diff
@@ -0,0 +1,436 @@
+
+### Verify available IP addresses
+
+EKS requires at least 2 subnets, each in a different availability zone and having 5 free IP addresses. 
+
+To verify that your subnets have enough IP addresses to upgrade the cluster you can run the following command:
+
+```
+CLUSTER=<cluster name>
+aws ec2 describe-subnets --subnet-ids \
+  $(aws eks describe-cluster --name ${CLUSTER} \
+  --query 'cluster.resourcesVpcConfig.subnetIds' \
+  --output text) \
+  --query 'Subnets[*].[SubnetId,AvailabilityZone,AvailableIpAddressCount]' \
+  --output table
+
+----------------------------------------------------
+|                  DescribeSubnets                 |
++---------------------------+--------------+-------+
+|  subnet-067fa8ee8476abbd6 |  us-east-1a  |  8184 |
+|  subnet-0056f7403b17d2b43 |  us-east-1b  |  8153 |
+|  subnet-09586f8fb3addbc8c |  us-east-1a  |  8120 |
+|  subnet-047f3d276a22c6bce |  us-east-1b  |  8184 |
++---------------------------+--------------+-------+
+```
+
+The [VPC CNI Metrics Helper](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/cmd/cni-metrics-helper/README.md) may be used to create a CloudWatch dashboard for VPC metrics. 
+
+### Verify EKS IAM role
+
+To verify that the IAM role is available and has the correct assume role policy in your account you can run the following commands:
+
+```
+CLUSTER=<cluster name>
+ROLE_ARN=$(aws eks describe-cluster --name ${CLUSTER} \
+  --query 'cluster.roleArn' --output text)
+aws iam get-role --role-name ${ROLE_ARN##*/} \
+  --query 'Role.AssumeRolePolicyDocument'
+  
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "eks.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+
+### Verify EKS security group
+
+To verify that the security groups exist in your account you can run the following commands:
+
+```
+CLUSTER=<cluster name>
+aws ec2 describe-security-groups \
+  --group-ids $(aws eks describe-cluster \
+    --query 'cluster.resourcesVpcConfig.securityGroupIds[*]' \
+    --name ${CLUSTER} --output text)
+    
+{                                                                                                                                      
+    "SecurityGroups": [                                                                                                                
+        {                                                                                                                              
+            "Description": "Communication between the control plane and worker nodegroups",                                            
+            "GroupName": "eksctl-prefix-cluster-ControlPlaneSecurityGroup-HFO1JTSFWL1J",
+```
+
+If you see output like the following then you should double check the security groups used by your cluster and available in your account.
```

Comment:
> then you should double check the security groups used by your cluster

But say even if they check double check and figure out that they deleted because EC2 API does indicate that they deleted, this should state what they should do eg recreate the cluster.